grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1063,6 +1063,16 @@ class StockMovementService {
                 createPicklist(stockMovementItem)
             }
         }
+        for (StockMovementItem stockMovementItem : stockMovement.lineItems) {
```

Comment:
I think it may be best to move this to createPicklist(RequisitionItem requisitionItem), you can add a parameter to it and to createPicklist(StockMovementItem stockMovementItem), it could be named validateQtyAvailable and if it's true you could check if there are any available items for qiven requisition item and throw the validation error then